### DISCUSSION TITLE
Image Occlusion ?🕵️

### DISCUSSION THREAD

**Discussion by eMerzh** [h16nning/skola#16] [Q&A]
Hello here :) 

I was a bit frustrated by anki, then i start creating a SR soft using TS, mantine, and supabase 😅  ....
while searching for a srs algorithm for this, i stumble on this ... nice discovery :) well done ! 👏 

AAAAnyways... discovered you support image occlusion but i couldn't make it work 🤔  i put the image but couldn't find how to add occlusion.

**Comment by eMerzh** [comment]
ooooh 
reading 
>What doesn't work (yet)?

in the readme ....  🤦  sooorrryyy

**Comment by h16nning** [comment]
Hello! First of all, sorry for the unclear labelling of the card type selection labels. I suppose this caused the confusion. At the moment only Normal (answer → question) and double sided cards are properly implemented. I was working on cloze cards. Image Occlusion is definitively a planned feature for the future. However, this feature is not my top priority at the moment as I think it is quite complex to implement. I was thinking about using a svg overlay just like anki does. However, I have relatively little experience with svg. As Skola is (only) a hobby project for me next to med school it may take some time until I get there. I hope you understand :)

With that being said, I'm glad you like the project. In fact, I started it because I was frustrated with Anki too...

The spaced repetition algorithm Skola uses is very similar to SM2.

**Reply by eMerzh** [reply]
Hehe thanks for the precisions :) 

Totally understand your view :) 
What makes me 🤨  is that i didn't read the README 😅  ... so i thought it was implemented, but I couldn't find the button :p 

For image occulusion, it's not THAT big of a deal for me, but my kid had to use it recently so i looked at it... 
Not sure about the svg part, why not... it could also be regular divs i guess...

I'll look at the project and see if i can do some stuff :)
